src/components/UserProfile/VolunteeringTimeTab/VolunteeringTimeTab.jsx

```diff
@@ -100,19 +99,15 @@ const WeeklyCommittedHours = props => {
       // Check if Value is greater than total hours in one week
       alert(`You can't commit more than ${MAXIMUM_WEEK_HOURS} hours per week.`);
       if (value === MAXIMUM_WEEK_HOURS + 1) {
-        props.setUserProfile({ ...props.userProfile, weeklyComittedHours: MAXIMUM_WEEK_HOURS });
-        props.setChanged(true);
+        props.setUserProfile({ ...props.userProfile, weeklycommittedHours: MAXIMUM_WEEK_HOURS });
       } else {
```

Comment:
is this else necessary?

Reply:
I just deleted it 😃

Reply:
Just to be sure, people with zero hours are treated specially on the Leaderboard --> they don't contribute to the "HGN Totals:" This change isn't going to break that is it?

Reply:
@one-community It won't

Reply:
@KaixiangGU: Just wanted to confirm that the backend property name is weeklycommittedHours and not weeklyComittedHours.
Logically the code looks good to me.
Also, I would like you to add 1-2 code comments wherever possible.

Reply:
@Aishwaryak01 Hi, I double-checked the backend, it's "weeklycommittedHours". Will add few comments.

Reply:
Sounds great!

Reply:
@Aishwaryak01 Added, please review again, thank you!